src/Components/ChangePassword/ChangePassword.tsx

```diff
@@ -1,31 +1,26 @@
 import { useForm } from "react-hook-form";
-// import logo from "../../../src/assets/bg-changepass.jpg";
 import axios from "axios";
 import { useNavigate } from "react-router-dom";
-import { ToastContainer, toast } from "react-toastify";
+import { toast } from "react-toastify";
+import { useContext } from "react";
+import { AuthContext } from "../../Context/AuthContext";
 
-export default function ChangePass({ handleClose }:any) {
+export default function ChangePass({ handleClose }: any) {
   const navigate = useNavigate();
 
+  let { reqHeaders, baseUrl } = useContext(AuthContext);
   const {
     register,
     handleSubmit,
     formState: { errors },
   } = useForm();
 
-  const onSubmit = (data:any) => {
+  const onSubmit = (data: any) => {
     // console.log(data);
-
     axios
-      .put(
-        "https://upskilling-egypt.com/api/v1/Users/ChangePassword",
-        data,
-        {
-          headers: {
-            Authorization: `Bearer ${localStorage.getItem("adminToken")}`,
-          },
-        }
-      )
+      .put(`${baseUrl}/Users/ChangePassword`, data, {
+        headers: reqHeaders,
+      })
       .then((response) => {
         handleClose();
         toast.success("password changed successsfully", {
```

Comment:
you need to provide the message from the backend first and then default to the message "password changed successfully" if no message is provided.

toast.success( response?.data?.message || "password changed successsfully"

but you need check the response structure and where message attribute is

Reply:
Also, there is something wrong with the toast component, and it doesn't show any toast messages.
You need to look into this problem as well.